aten/src/ATen/Context.h

```diff
@@ -125,7 +126,13 @@ class CAFFE2_API Context {
   bool deterministic_cudnn = false;
   bool benchmark_cudnn = false;
   bool enabled_mkldnn = true;
-  at::QEngine quantized_engine = at::QEngine::FBGEMM;
+  at::QEngine quantized_engine =
+#ifdef USE_PYTORCH_QNNPACK
```

Comment:
both fbgemm and qnnpack can be present on server side - we should give fbgemm the priority